test_verify_new_cbp_creation_not_blocked_by_invalid_cbp failing in external mode

Test case is failing in external mode cluster:
Test case : tests/manage/pv_services/test_verify_new_cbp_creation_not_blocked_by_invalid_cbp.py::test_verify_new_cbp_creation_not_blocked_by_invalid_cbp

Error:

```
AssertionError: Ceph Block Pool cbp-test-8e5d7b18e73a40159aee1179ec9b90f is not created.
assert False
 +  where False = <function verify_block_pool_exists at 0x7f41985a5d30>('cbp-test-8e5d7b18e73a40159aee1179ec9b90f')
 +    where <function verify_block_pool_exists at 0x7f41985a5d30> = helpers.verify_block_pool_exists
 +    and   'cbp-test-8e5d7b18e73a40159aee1179ec9b90f' = <ocs_ci.ocs.resources.ocs.OCS object at 0x7f41667374f0>.name
```

--- a/tests/manage/pv_services/test_verify_new_cbp_creation_not_blocked_by_invalid_cbp.py
+++ b/tests/manage/pv_services/test_verify_new_cbp_creation_not_blocked_by_invalid_cbp.py
@@ -1,7 +1,12 @@
 import logging
 
 from ocs_ci.helpers import helpers
-from ocs_ci.framework.testlib import polarion_id, skipif_ocs_version, tier2
+from ocs_ci.framework.testlib import (
+    polarion_id,
+    skipif_ocs_version,
+    tier2,
+    skipif_external_mode,
+)
 
 log = logging.getLogger(__name__)
 
@@ -9,6 +14,7 @@ log = logging.getLogger(__name__)
 @tier2
 @skipif_ocs_version("<4.3")
 @polarion_id("OCS-2130")
+@skipif_external_mode
 def test_verify_new_cbp_creation_not_blocked_by_invalid_cbp(teardown_factory):
     """
     Test to verify new ceph block pool can be created without deleting

